demo/app/components/editor-layer/circles-editor-layer/circles-editor-layer.component.ts

```diff
@@ -2,63 +2,90 @@ import { Component, OnInit } from '@angular/core';
 import { CircleEditorObservable } from '../../../../../src/angular-cesium-entities-editor/models/circle-editor-observable';
 import { CircleEditUpdate } from '../../../../../src/angular-cesium-entities-editor/models/circle-edit-update';
 import { CirclesEditorService } from '../../../../../src/angular-cesium-entities-editor/services/entity-editors/circles-editor/circles-editor.service';
+import { LabelProps } from '../../../../../src/angular-cesium-entities-editor/models/label-props';
 
 @Component({
-	selector : 'circles-editor-layer',
-	templateUrl : 'circles-editor-layer.component.html',
-	styleUrls : ['./circles-editor-layer.component.css']
+  selector: 'circles-editor-layer',
+  templateUrl: 'circles-editor-layer.component.html',
+  styleUrls: ['./circles-editor-layer.component.css']
 })
 export class CirclesEditorLayerComponent implements OnInit {
-	
-	editing$: CircleEditorObservable;
-	enableEditing = true;
-	
-	constructor(private circlesEditor: CirclesEditorService) {
-	}
-	
-	ngOnInit(): void {
-	}
-	
-	startEdit() {
-		if (this.editing$) {
-			this.stopEdit();
-		}
-		this.editing$ = this.circlesEditor.create({allowDrag: true});
-		this.editing$.subscribe((editUpdate: CircleEditUpdate) => {
+
+  editing$: CircleEditorObservable;
+  enableEditing = true;
+
+  constructor(private circlesEditor: CirclesEditorService) {
+  }
+
+  ngOnInit(): void {
+  }
+
+  startEdit() {
+    if (this.editing$) {
+      this.stopEdit();
+    }
+    this.editing$ = this.circlesEditor.create({ allowDrag: false });
+    this.editing$.subscribe((editUpdate: CircleEditUpdate) => {
       console.log(editUpdate);
-		});
-	}
-	
-	stopEdit() {
+    });
+  }
+
+  stopEdit() {
     if (this.editing$) {
       this.editing$.dispose();
       this.editing$ = undefined;
     }
-	}
+  }
 
-	editFromExisting() {
-		if (this.editing$) {
-			this.stopEdit();
-		}
-		this.editing$ = this.circlesEditor.edit(Cesium.Cartesian3.fromDegrees(-70, 0), 800000);
-	}
+  editFromExisting() {
+    if (this.editing$) {
+      this.stopEdit();
+    }
+    this.editing$ = this.circlesEditor.edit(Cesium.Cartesian3.fromDegrees(-70, 0), 800000);
+    this.editing$.setLabelsRenderFn((update: CircleEditUpdate) => {
+      const newLabels: LabelProps[] = [];
+      newLabels.push(
+        {
+          text: 'Center',
+          scale: 0.6,
+          eyeOffset: new Cesium.Cartesian3(10, 10, -1000),
+          fillColor: Cesium.Color.BLUE,
+        },
+        {
+          text: Math.round(update.radius).toString(),
+          scale: 0.6,
+          eyeOffset: new Cesium.Cartesian3(10, 10, -1000),
+          fillColor: Cesium.Color.RED,
+        },
+      );
+      return newLabels;
+    });
+    setTimeout(() =>
+      this.editing$.updateLabels((update: CircleEditUpdate, labels) => {
```

Comment:
why you need to pass a func?
you can do `this.editing$.updateLabels(labels);`
and if you want the current positions you can do `const positions = editing$.getPositions`